### DISCUSSION TITLE
Support for Espeasy

### DISCUSSION THREAD

**Discussion by jbaream** [Bodmer/TFT_eSPI#2605] [General]
Hi, there is a firmware called Espeasy (https://github.com/letscontrolit/ESPEasy/) suited for ESP8266 and ESP32 MCUs which allows to have a multisensor device in a very short period of time for normal users without programmer skills, since it has a web-based configuration and is based in a plugin arquitecture, among many other great capabilities. Some of the TFT supported here in TFT_eSPI are supported in EspEasy through a couple of plugins, using Adafruit libraries. Then you can choose your TFT model at runtime, choosing the appropriate plugin. It would be nice if a plugin based in TFT_eSPI could be developed, but unfortunately this library doesn't allow to choose the TFT model at runtime, rather it have be choosen at compilation time, if I understood well. Mi question is: can  TFT_eSPI be adapted to a new library where the TFT model can be choosen at runtime and, if so, how easily? 

Cheers!

Jose.

**Comment by Bodmer** [comment]
Unfortunately the library has not been created with that use case in mind. It would be possible to alter the library but this is not something I have time to help with.
